build.gradle

```diff
@@ -3,88 +3,60 @@ buildscript {
     jcenter()
     maven { url "https://spinnaker.bintray.com/gradle" }
     maven { url "https://plugins.gradle.org/m2/" }
+    if (spinnakerGradleVersion.endsWith("-SNAPSHOT")) {
+      mavenLocal()
+    }
   }
   dependencies {
-    classpath 'com.netflix.spinnaker.gradle:spinnaker-dev-plugin:5.2.2'
+    classpath "com.netflix.spinnaker.gradle:spinnaker-dev-plugin:$spinnakerGradleVersion"
+    if (Boolean.valueOf(enablePublishing)) {
```

Comment:
you'll need to update the travis scripts in the gradle directory to pass -PenablePublishing=true instead of -I gradle/init-publish.gradle